packages/core/src/common/keybinding.spec.ts

```diff
@@ -192,39 +193,69 @@ describe("keys api", () => {
         expect(keycode).is.undefined;
     });
 
-    if (isOSX) {
-        it("should parse a string containing ctrl/option to a KeyCode correctly (OS X)", () => {
-            const keycodeCtrl = KeyCode.parse("ctrl+b");
-            expect(keycodeCtrl).is.not.undefined;
-            if (keycodeCtrl) {
-                expect(keycodeCtrl.meta).to.be.true;
-                expect(keycodeCtrl.key).is.equal(Key.KEY_B);
-            }
+    it("should parse a string containing special modifiers to a KeyCode correctly", () => {
+        const stub = sinon.stub(os, 'isOSX').value(false);
 
-            const keycodeOption = KeyCode.parse("option+b");
-            expect(keycodeOption).is.not.undefined;
-            if (keycodeOption) {
-                expect(keycodeOption.alt).to.be.true;
-                expect(keycodeOption.key).is.equal(Key.KEY_B);
-            }
-        });
-    } else {
-        it("should parse a string containing ctrl/option to a KeyCode correctly", () => {
-            const keycode = KeyCode.parse("ctrl+b");
-            expect(keycode).is.not.undefined;
-            if (keycode) {
-                expect(keycode.ctrl).to.be.true;
-                expect(keycode.key).is.equal(Key.KEY_B);
-            }
+        const keycode = KeyCode.parse("ctrl+b");
+        expect(keycode).is.not.undefined;
+        if (keycode) {
+            expect(keycode.ctrl).to.be.true;
+            expect(keycode.key).is.equal(Key.KEY_B);
+        }
 
-            const keycodeOption = KeyCode.parse("option+b");
-            expect(keycodeOption).is.not.undefined;
-            if (keycodeOption) {
-                expect(keycodeOption.alt).to.be.true;
-                expect(keycodeOption.key).is.equal(Key.KEY_B);
-            }
-        });
-    }
+        const keycodeOption = KeyCode.parse("option+b");
+        expect(keycodeOption).is.not.undefined;
```

Comment:
This should be undefined on other platforms than macOS, right?
This make me think that we need a AltOrOption...

Reply:
@MarkZ3 hmmm good point, I thought option was mapped one to one to alt, isn't that the case? Or you want something more explicit?

Reply:
I think maybe it doesn't matter for now. Later we might want to think how to handle how the "defaults" are displayed.